optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -0,0 +1,670 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2017
+# Author: Cleber Rosa <crosa@redhat.com>
```

Comment:
(PS: I know it's not a public API and we do not guarantee to keep it, but still we know it's been in use so we should at least try. If that seems no worth the effort we should at least sketch the final version and create a card to work on it...)